### PR TITLE
CBG-575: Code coverage for base package

### PR DESCRIPTION
**Pull request by sarathkumarsivan**
- Added coverage for TestLoggingLevel
- Code cleanup

### CODE REVIEW

File: base/logging_test.go [hunk 1]
```diff
@@ -148,3 +149,93 @@ func BenchmarkLogRotation(b *testing.B) {
...
+	level = Level(0x5)
+	assert.Equal(t, logging.NONE, level.cgLevel())
+	assert.Equal(t, "Level(5)", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling unknown logging level")
+	assert.Error(t, level.UnmarshalText([]byte(level.String())), level.String())
+}
+
+func TestColor(t *testing.T) {
```
Comment (bbrks) [inline base/logging_test.go]: This test works OK on macOS and Linux, but due to the fact that these ANSI colors are not cross-platform, `colorEnabled()` _always_ returns false on `GOOS=windows` and `$TERM=dumb`, which will make this test fail when run on these platforms.

If we really want coverage, we could skip the test for the above reason when `colorEnabled()` returns false, even after you set `consoleLogger.ColorEnabled`.

Comment (bbrks) [inline base/logging_test.go]: Rename test to be logging specific.

```suggestion
func TestLogColor(t *testing.T) {
```

**Review by bbrks — CHANGES_REQUESTED** [review]
Looks fine, mostly old code being covered - but the color test won't work cross-platform.

File: base/logging_test.go [hunk 2]
```diff
@@ -148,3 +149,96 @@ func BenchmarkLogRotation(b *testing.B) {
...
+	assert.Equal(t, logging.NONE, level.cgLevel())
+	assert.Equal(t, "Level(5)", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling unknown logging level")
+	assert.Error(t, level.UnmarshalText([]byte(level.String())), level.String())
+}
+
+func TestLogColor(t *testing.T) {
+	consoleLogger.ColorEnabled = false
```
Comment (bbrks) [inline base/logging_test.go]: Shouldn't this be true?

```suggestion
	consoleLogger.ColorEnabled = true
```

**Review by bbrks — CHANGES_REQUESTED** [review]
We're never running the assertions in the changed part of TestLogColor.

File: base/logging_test.go (hunk 2 above)
Comment (sarathkumarsivan) [reply base/logging_test.go]: Yeah, you're right! This has been fixed in the latest revision.

**Review by bbrks — APPROVED** [review]
Looks OK to me 👍

**Pull request merged**